src/runtime/vortex/runtime/onnx/onnxruntime.py

```diff
@@ -82,6 +82,25 @@ def predict(self, *args, **kwargs) -> np.ndarray :
         )
         return outputs[0]
 
+    @staticmethod
+    def resize_batch(images : List[np.ndarray], size : Tuple[int,int,int,int], resize_kind='stretch') :
```

Comment:
enlight me why this is declared again in onnxruntime?

Reply:
as the comment said, "this is the same as BaseRuntime implementation, apart from h,w is flipped"
this is required to make non-square input possible.

Reply:
the short explanation is, `cv2.resize()` uses a flipped representation for the argument you give than the actual array size ((w,h) in argument, (h,w) in actual array).   
Then the onnx exporter get the input metadata from the example input given to it when exporting. So when we pull the metadata on runtime, this has a flipped value in it (h,w). So the original resize will have it flipped as well (the argument is (h,w) so the actual array is (w,h)). But the model expect the input array size to have a (h,w) input.  
So that is why it need to be flipped

Reply:
okay, in the meantime pls resolve the conflict

Reply:
done